app/controllers/users_controller.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  def index
```

Comment:
"Aki nem admin az nem látja az admin kezelő felületet" Azt bárki mondhassa!

Ha a `users#index` az admin kezelő felület, akkor a mostani kód alapján el tudom érni egy sima user-rel is. Próbáld ki, hogy csak beírod az url-t.

Szükségünk lenne ennél az endpontnál is egy policy-ra, amit authorize-zal meghívhatunk.

Itt ugye nincs konkrét rekord, amin authorize-t tudunk hívni, ezért átadhatunk szimbólumot is.

```ruby
# így csináltuk a többi kontroller akcióban, ahol volt konkrét rekord, amit authorizáltunk
authorize MyFavouriteModel.find_by(id: params[:id])

# ha nincs konkrét rekordom, de meg szeretném hívni az authorize-t, akkor tudok ilyet csinálni
authorize :my_favourite_model
```